cli/portforward.go

```diff
@@ -180,6 +181,13 @@ func (r *RootCmd) portForward() *clibase.Cmd {
 			Description: "Forward UDP port(s) from the workspace to the local machine. The UDP connection has TCP-like semantics to support stateful UDP protocols.",
 			Value:       clibase.StringArrayOf(&udpForwards),
 		},
+		{
+			Flag:        "disable-autostart",
+			Description: "Disable starting the workspace automatically when connecting via port forward.",
+			Env:         "CODER_PORT_FORWARD_DISABLE_AUTOSTART",
```

Comment:
It'd be nice to have a global `CODER_DISABLE_AUTOSTART` so you don't have to define it for all subcommands separately, not sure if we want _only_ a global one, perhaps there's a case for having both. 🤔

This might require a deeper think about how this and related flags should behave.